src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java

```diff
@@ -33,128 +41,19 @@
-                String valueSampa = csvRecord.get("value_sampa");
-                sound.setValueSampa(valueSampa);
-                
-                boolean diacritic = Boolean.valueOf(csvRecord.get("diacritic"));
-                sound.setDiacritic(diacritic);
-                
-                if (StringUtils.isNotBlank(csvRecord.get("sound_type"))) {
-                    SoundType soundType = SoundType.valueOf(csvRecord.get("sound_type"));
-                    sound.setSoundType(soundType);
-                }
-                
-                Integer usageCount = Integer.valueOf(csvRecord.get("usage_count"));
-                sound.setUsageCount(usageCount);
-                
-                sounds.add(sound);
-            }
-        } catch (IOException ex) {
-            logger.error(ex);
-        }
-        
-        return sounds;
-    }
-    
-    /**
-     * For information on how the CSV files were generated, see {@link LetterCsvExportController#handleRequest}.
-     */
-    public static List<Letter> getLettersFromCsvBackup(File csvFile, SoundDao soundDao) {
-        logger.info("getLettersFromCsvBackup");
-        
-        List<Letter> letters = new ArrayList<>();
-        
-        Path csvFilePath = Paths.get(csvFile.toURI());
-        logger.info("csvFilePath: " + csvFilePath);
-        try {
-            Reader reader = Files.newBufferedReader(csvFilePath);
-            CSVFormat csvFormat = CSVFormat.DEFAULT
-                    .withHeader(
-                            "id",
-                            "text",
-                            "diacritic",
-                            "usage_count"
-                    )
-                    .withSkipHeaderRecord();
-            CSVParser csvParser = new CSVParser(reader, csvFormat);
-            for (CSVRecord csvRecord : csvParser) {
-                logger.info("csvRecord: " + csvRecord);
-                
-                Letter letter = new Letter();
-                
-                String text = csvRecord.get("text");
-                letter.setText(text);
 
-                boolean diacritic = Boolean.valueOf(csvRecord.get("diacritic"));
-                letter.setDiacritic(diacritic);
-                
-                Integer usageCount = Integer.valueOf(csvRecord.get("usage_count"));
-                letter.setUsageCount(usageCount);
-                
-                letters.add(letter);
-            }
-        } catch (IOException ex) {
-            logger.error(ex);
-        }
-        
-        return letters;
-    }
-    
     /**
      * For information on how the CSV files were generated, see {@link LetterSoundCorrespondenceCsvExportController#handleRequest}.
      */
     public static List<LetterSoundCorrespondence> getLetterSoundCorrespondencesFromCsvBackup(File csvFile, LetterDao letterDao, SoundDao soundDao, LetterSoundCorrespondenceDao letterSoundCorrespondenceDao) {
         logger.info("getLetterSoundCorrespondencesFromCsvBackup");
-        
+
```

Comment:
Is this replacing spaces with tabs?

Reply:
@jo-elimu removed all spaces/tabulation symbols from the line